<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenetrePrincipale</class>
 <widget class="QMainWindow" name="FenetrePrincipale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Viginond 57</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>768</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,3">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_requete">
      <property name="title">
       <string>Requêtes utilisateur</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_requete">
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_telechargement">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Téléchargement des données Vigicrues</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QTextBrowser" name="textBrowser_vigicrues"/>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_proxy">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="title">
          <string>Gestion du proxy internet</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_proxy">
          <item row="0" column="0">
           <widget class="QRadioButton" name="radioButton_proxy_ddt">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>DDT</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="radioButton_proxy_aucun">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Aucun</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="radioButton_proxy_prefecture">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Préfecture</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QRadioButton" name="radioButton_proxy_autre">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Configuration</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QTextBrowser" name="textBrowser_internet">
         <property name="minimumSize">
          <size>
           <width>280</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>23</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QGroupBox" name="groupBox_bassin_versant">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>120</height>
          </size>
         </property>
         <property name="title">
          <string>Bassins versants disponibles pour étude</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout_bassin_versant"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_visualisation">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Visualisation des crues du bassin versant</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_visualisation" rowstretch="1,1">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget_graphique">
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTabWidget" name="tabWidget_tableau"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
